config/v1/types_infrastructure.go

```diff
@@ -1035,6 +1042,18 @@ type BareMetalPlatformStatus struct {
 	// +optional
 	LoadBalancer *BareMetalPlatformLoadBalancer `json:"loadBalancer,omitempty"`
 
+	// DNSRecordsType determines whether records for api, api-int, and ingress
+	// are provided by the internal DNS service or externally. `Internal` configures
+	// DNS records in the internal service. `External` means no records will be
```

Comment:
Internal allows DNS resolution for components within the cluster right? It's configuring coredns?